README.md

```diff
@@ -1,2 +1,7 @@
-# CHSH
+# Bell Inequality on a Quantum computer
```

Comment:
What is a bell inequality?

Reply:
(https://en.wikipedia.org/wiki/CHSH_inequality)